localstack/utils/bootstrap.py

```diff
@@ -184,7 +184,13 @@ def get_docker_container_names():
 
 
 def get_main_container_ip():
-    cmd = "docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' %s" % config.MAIN_CONTAINER_NAME
+    container_name = get_main_container_name()
+    cmd = "docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' %s" % container_name
+    return run(cmd).strip()
+
+
+def get_main_container_name():
+    cmd = "docker inspect -f '{{ .Name }}' %s | cut -c 2-" % config.HOSTNAME
```

Comment:
Instead of piping to `cut` here, can we do the string manipulation directly in Python - something like:
```
cmd = "docker inspect -f '{{ .Name }}' %s" % config.HOSTNAME
return run(cmd).strip().lstrip('/')
```

Reply:
yes @whummer! great point 👍  I just pushed it :)